common/source/docs/common-compass-calibration-in-mission-planner.rst

```diff
@@ -103,7 +107,7 @@ Older versions of ArduPilot rely on the ground station to calculate the compass
 
       Mission Planner: Live Compass Calibration
 
-- The aim is to rotate the vehicle so that the coloured trail hits each of the white dots.  One way to do this is to hold the vehicle in the air and rotate it slowly so that each side (front, back, left, right, top and bottom) points down towards the earth for a few seconds in turn.
+- The aim is to rotate the vehicle so that the colored trail hits each of the white dots.  One way to do this is to hold the vehicle in the air and rotate it slowly so that each side (front, back, left, right, top and bottom) points down towards the earth for a few seconds in turn.
```

Comment:
a battle of american english vs british english! :-)  being Canadian I'd leave it with the "u" in there but it's correct either way and you're doing the majority of the typing these days.

Reply:
my dictionary lol